### DISCUSSION TITLE
是否有QQ微信交流群？

### DISCUSSION THREAD

**Discussion by mumu-sild** [kanition/pbrtbook#171] [其他]
本人微信：y870744511
以学习图形两年有余，也想在动漫领域有所建树，想找到志同道合的伙伴

**Comment by kanition** [comment]
我个人比较内向，也自觉比较欠缺管理能力，所以目前没有建过这样的群（也担心本项目和我生活中的社交账号发生关联破坏了匿名署名承诺）。

之前好像有人问过一次两次，我说我没建过后也就没有后续了。如果题主有意愿的话可以自己建一个，也欢迎来这里宣传你建的群，我说不定会假装路人加进去（悄悄）。

还有一种办法是常来本项目讨论区，直接发帖提问，这里人才那么多，我超~喜欢这里的（逃）。之前有位同好因为第五章的内容在issue区和我~~吵架~~讨论了几十层，虽然学术观点不同，但我喜欢这种充分的交流。

**Comment by kanition** [comment]
想起来题主既然学过两年了，应该比我学得深入得多。本项目虽然是2021年开始的，但最近一两年工作原因花的时间少得多，实际投入时间总和可能不到一年。题主如果发现本书存在错误，欢迎及时反馈。

**Reply by mumu-sild** [reply]
我也大概是2021年开始学习的，但离线渲染方向也是近期才开始重点学习，还是有很多要向你多多学习。
我创建了一个QQ群：734217395 离线渲染pbrt学习交流群，希望有更多的人可以一起因为兴趣相投相聚。

QQ群：734217395

**Comment by mumu-sild** [comment]
QQ群：734217395
